src/transformStateWithClones.js

```diff
@@ -8,6 +8,39 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const copyState = Object.assign({}, state);
+  const arrayOfversions = [];
```

Comment:
Don't use data type name in variable name
```suggestion
  const arrayOfVersions = [];
```